packages/l10n/src/country-information.ts

```diff
@@ -1,8 +1,8 @@
-import PropTypes from 'prop-types';
+import * as PropTypes from 'prop-types';
 import createL10NInjector from './create-l10n-injector';
 import { getSupportedLocale, mapLocaleToIntlLocale } from './utils';
 
-const getImportChunk = locale => {
+const getImportChunk = (locale: string): Promise<any> => {
```

Comment:
We should avoid using `any`. Try

```suggestion
const getImportChunk = (locale: string): Promise<Record<string, string>> => {
```